appveyor.yml

```diff
@@ -73,8 +89,14 @@ artifacts:
 #on_finish:
 #  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
+
 # on build failure
 on_failure:
+- ps: |
+    Get-ChildItem -recurse artifacts\log\*.binlog -ErrorAction SilentlyContinue `
+    | ForEach-Object {
+        Push-AppveyorArtifact "$_"
+    }
```

Comment:
Doesn't it publish by this?
![image](https://user-images.githubusercontent.com/4403806/80366394-8c7e1300-88cc-11ea-8d80-4c5dc91aaf5f.png)

Reply:
No it does not `on_failure:`.